tapdb/universe_stats.go

```diff
@@ -155,7 +155,7 @@ type assetEventsCache = *lru.Cache[eventQuery, cachedAssetEvents]
 
 // statsQueryCacheSize is the total number of asset query responses that we'll
 // hold inside the cache.
-const statsQueryCacheSize = 80_000
+const statsQueryCacheSize = 200_000
```

Comment:
At 200k, I think the standby memory won't exceed 10 Mb (based on the cache key data structure size).